packages/hoc-input/src/formInput.js

```diff
@@ -140,6 +142,12 @@ const formInput = WrappedComponent => {
       );
     }
 
+    componentDidMount() {
+      if (this.props.focused) {
+        this.setInputFocus();
```

Comment:
![2018-07-03 08 58 42](https://user-images.githubusercontent.com/17936244/42218643-ac7679ce-7e9f-11e8-84d2-4a216c394811.gif)

Reply:
IDK if the bug is on this part of the code

Reply:
What's the expected behavior?

Reply:
https://cravefood.github.io/farmblocks/?selectedKind=Search%20Field&selectedStory=update%20items&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel